tomcat/src/main/java/org/apache/coyote/http11/Http11Processor.java

```diff
@@ -28,12 +37,22 @@ public void run() {
     public void process(final Socket connection) {
         try (final var inputStream = connection.getInputStream();
              final var outputStream = connection.getOutputStream()) {
+            BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(inputStream));
```

Comment:
```suggestion
        try (final var inputStream = connection.getInputStream();
             BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(inputStream));
             final var outputStream = connection.getOutputStream()) {
```

`BufferedReader` 역시 `Closable` 을 구현하는 클래스입니다. 
자원 사용이 끝난 후 자원을 반납하는 것을 잊지 않도록 try-with-resource 안에 포함시켜 볼까요?

Reply:
오.. close 해야 한다는 것을 완전히 잊고 있었네요..
이래서 try-with-resource를 사용하는 것이군요!! 👍👍👍